src/pages/workspace/WorkspaceInvitePage.js

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {TextInput, View} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import Str from 'expensify-common/lib/str';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import Navigation from '../../libs/Navigation/Navigation';
+import styles from '../../styles/styles';
+import Text from '../../components/Text';
+import Button from '../../components/Button';
+import compose from '../../libs/compose';
+import ONYXKEYS from '../../ONYXKEYS';
+import {invite} from '../../libs/actions/Policy';
+import TextLink from '../../components/TextLink';
+import getEmailKeyboardType from '../../libs/getEmailKeyboardType';
+import themeColors from '../../styles/themes/default';
+import Growl from '../../libs/Growl';
+import CONST from '../../CONST';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+
+    /** The policy passed via the route */
+    policy: PropTypes.shape({
+        /** The policy name */
+        name: PropTypes.string,
+    }),
+
+    /** URL Route params */
+    route: PropTypes.shape({
+        /** Params from the URL path */
+        params: PropTypes.shape({
+            /** policyID passed via route: /workspace/:policyID/invite */
+            policyID: PropTypes.string,
+        }),
+    }).isRequired,
+};
+
+const defaultProps = {
+    policy: {
+        name: '',
+    },
+};
+
+class WorkspaceInvitePage extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            emailOrPhone: '',
+            welcomeNote: '',
+        };
+
+        this.getWelcomeNotePlaceholder = this.getWelcomeNotePlaceholder.bind(this);
```

Comment:
NAB, I think we do not need to bind this since `inviteUser()` already has it's `this` bound to the component and the only other usage is called in the `render()`.